tests/imap/MailCollector.php

```diff
@@ -41,31 +41,33 @@ class MailCollector extends DbTestCase {
    private $mailgate_id;
 
    public function testGetEmpty() {
-      $this
-         ->if($this->newTestedInstance)
-         ->then
-            ->array($this->testedInstance->fields)
-               ->isIdenticalTo([])
-            ->boolean($this->testedInstance->getEmpty())
-            ->array($this->testedInstance->fields)
-               ->isIdenticalTo([
-                  'id'              => '',
-                  'name'            => '',
-                  'host'            => '',
-                  'login'           => '',
-                  'filesize_max'    => '2097152',
-                  'is_active'       => 1,
-                  'date_mod'        => '',
-                  'comment'         => '',
-                  'passwd'          => '',
-                  'accepted'        => '',
-                  'refused'         => '',
-                  'use_kerberos'    => '',
-                  'errors'          => '',
-                  'use_mail_date'   => '',
-                  'date_creation'   => '',
-                  'requester_field' => ''
-               ]);
+      $this->newTestedInstance();
+
+      $this->array($this->testedInstance->fields)->isIdenticalTo([]);
+      $this->boolean($this->testedInstance->getEmpty())->isTrue();
+
+      $fields = $this->testedInstance->fields;
+      ksort($fields);
```

Comment:
To prevent failure on ksort.
```suggestion
      $this->array($fields);
      ksort($fields);
```

Reply:
done